src/App.js

```diff
@@ -6,12 +6,20 @@ import { Provider } from "react-redux";
 import Routes from "./routes/index";
 import history from "./utils/history";
 import store from "./store/index";
+import QRCodeGenerator from "../src/components/QRCodeGenerator";
+import QrCodeScanner from "./view/QrCodeScanner";
+
 
 function App() {
   return (
     <Provider store={store}>
       <div className="App">
         <Routes history={history} />
+        <QRCodeGenerator/>
```

Comment:
Where is react router configured in this file?
https://v5.reactrouter.com/web/example/route-config